workspace-manager-client/build.gradle

```diff
@@ -1,32 +1,36 @@
+buildscript {
+	apply from: "../common.gradle"
+}
+
 plugins {
 	id "java-library"
 	id "maven-publish"
 
 	id "com.jfrog.artifactory" version "4.18.2"
-	id "org.hidetake.swagger.generator"
+	id "org.hidetake.swagger.generator" version "2.18.2"
 }
 
+group = 'bio.terra'
+version = "${ext.theVersion}"
 sourceCompatibility = JavaVersion.VERSION_1_8 // Rawls consumes this and is Java 8
 
-repositories {
-	mavenCentral()
-}
-
 dependencies {
 	// Swagger deps, versions controlled by dependencyManagement in root project
-	implementation group: "org.glassfish.jersey.core", name: "jersey-client"
-	implementation group: "org.glassfish.jersey.media", name: "jersey-media-json-jackson"
-	implementation group: "org.glassfish.jersey.media", name: "jersey-media-multipart"
-	implementation group: "com.fasterxml.jackson.datatype", name: "jackson-datatype-jsr310"
-	implementation group: "io.swagger.core.v3", name: "swagger-annotations"
-	swaggerCodegen group: "io.swagger.codegen.v3", name: "swagger-codegen-cli"
+	implementation group: "org.glassfish.jersey.core", name: "jersey-client", version: "2.32"
+	implementation group: "org.glassfish.jersey.media", name: "jersey-media-json-jackson", version: "2.32"
+	implementation group: "org.glassfish.jersey.media", name: "jersey-media-multipart", version: "2.32"
+	implementation group: "com.fasterxml.jackson.datatype", name: "jackson-datatype-jsr310", version: "2.11.3"
```

Comment:
Why are we allowing these to get out of sync from the spring boot versions, rather than using `dependencyManagement` as before?

Reply:
I do not want to have client as a subproject of the service. One of the goals of the work is that clienttest should not have to build the service to get itself built.

Is there a reason why the dependencies are required to be in sync?

From what I read, dependencyManagement is being replace by a new gradle feature that is in preview in 7.0.

Reply:
The dependency management provided by the plugin is focused on keeping versions of dependencies in sync between multiple projects or subprojects, it doesn't add a dependency between projects itself.

The reason to keep them in sync would be to make sure that the json parsing for the client and the server is following the same logic and matches.

Reply:
Where would you put the dependencyManagement declaration in this scheme?

I don't think it can go in `settings.gradle`.  Putting it in top-level `build.gradle` doesn't make sense to me, because that is not used in the component builds. Putting it in `common.gradle` wouldn't seem to work, since that is expanded per-project so wouldn't give the desired identical results.